test/quality/.grype.yaml

```diff
@@ -1 +1,22 @@
 by-cve: true
+
+match:
+  java:
+    using-cpes: true
```

Comment:
Why are all these enabled?

Reply:
I had a conversation with @westonsteimel  about this, we want to be able to test this kind of change with as broad of a lens in terms of CPE matching as possible. I'll dig through the QG findings to see if there are additional changes needed. 

I'll also add a comment in this config as to why this is enabled.